### DISCUSSION TITLE
Dialogs can overflow the screen

### DISCUSSION THREAD

**Discussion by ghost** [IndiePython/nodezator#87] [Concerns/constructive criticism]
If I've enough errors the dialog can grow so large the OK button goes off-screen,

This could be worked around by the Enter key working for OK, but still isn't the best user experience. At this moment I've to restart the app.

![image](https://github.com/IndiePython/nodezator/assets/29675001/a8832598-84fc-4efd-a4a6-00a3736b7f46)

**Comment by KennedyRichard** [comment]
I apologize for this bad experience. Thank you for your patience. That fact that you only consider it cause for constructive criticism/concern is very kind of you. In my opinion it is actually a serious user experience problem that prevents or significantly hinders proper usage.

I'll create an issue for it and prioritize solving this problem. I expect the patch solving the problem to be released within a week or two. The problem is fairly easy to solve because all the services needed to do so are already implemented, I just have to put what is needed together. I'll probably make it so whenever the contents of dialogs are big enough to overflow the screen they are turned into a scrollable panel.

The only reason I'm estimating such time to solve it is because I also have to tend to other tasks in the meantime, some related and some unrelated to my open-source work.

As always, let me know if you need anything else. I'll keep replying to your other discussions/issues throughout the week as I find the time, thank you for the continued feedback and keep it coming.

Peace.

**Comment by ghost** [comment]
As an added note, this overflow issue can be witnessed even on longer titled notes. In [ComfyUI](https://github.com/comfyanonymous/ComfyUI) we can resize nodes to any arbitrary size (and btw, most added graphs and such are contained IN the node, not on it's side, that could be an idea).
